09-secrets.md

```diff
@@ -1,28 +1,33 @@
 ## Secrets top-level element
 
-Secrets are a flavour of Configs focusing on sensitive data, with specific constraint for this usage. As the platform implementation may significantly differ from Configs, dedicated Secrets section allows to configure the related resources.
+Compose provides a way for you to use secrets without having to use environment variables to store information. If you’re injecting passwords and API keys as environment variables, you risk unintentional information exposure. Environment variables are often available to all processes, and it can be difficult to track access. They can also be printed in logs when debugging errors without your knowledge. Using secrets mitigates these risks.
 
-The top-level `secrets` declaration defines or references sensitive data that can be granted to the services in this
-application. The source of the secret is either `file` or `external`.
+Secrets are also useful for locally replicating a remote development setup with Kubernetes. Using secrets makes your application scalable.
```

Comment:
I'm wondering if this type of content isn't more documentation one than specification one?
What we need on the specification level is an explanation of what a secret is, when or why using it. The current definition don't reach this goal, it's the same with this proposal, maybe we should take inspiration from [the Volumes part](https://github.com/compose-spec/compose-spec/blob/master/07-volumes.md) of the spec to provide a clearer definition of Secrets (and Configs?)

Reply:
Right. I think the right move here would be to have actual Compose docs, and have the Spec just be a Spec, otherwise concerns are mixed and the water gets a bit murky. There's a clear contrast between language elsewhere which just describes how a Compose file should be parsed and what a Compose implementation should do with that information vs this kind of language, and I don't think that information should reside within the Spec.